src/components/TokenBalance.test.tsx

```diff
@@ -98,6 +94,12 @@ describe('FiatExchangeTokenBalance and HomeTokenBalance', () => {
         },
       })
 
+      jest
```

Comment:
I think we can define this globally in the file and then only use it where needed in the tests to reduce duplication.